source/config/configFlags.py

```diff
@@ -256,15 +256,25 @@ def _displayStringLabels(self):
 class ParagraphStartMarker(DisplayStringStrEnum):
 	NONE = ""
 	SPACE = " "
+	# Translators: This is a paragraph start marker used in braille.
+	# The default symbol is the pilcrow,
+	# a symbol also known as "paragraph symbol" or "paragraph marker".
+	# This symbol should translate in braille via LibLouis automatically.
+	# If there is a more appropriate character for your locale,
+	# consider overwriting this (e.g. for Ge'ez ፨).
+	# You can also use Unicode Braille such as ⠘⠏.
+	# Ensure this is consistent with other strings with the context "paragraphMarker".
```

Comment:
this comment should be deleted now that it has been moved to getParagraphStartMarker